patches/minecraft/net/minecraft/entity/player/PlayerEntity.java.patch

```diff
@@ -425,7 +425,29 @@
        }
     }
  
-@@ -1703,7 +1818,10 @@
+@@ -1462,6 +1577,10 @@
+    }
+ 
+    public void func_195068_e(int p_195068_1_) {
++      net.minecraftforge.event.entity.player.PlayerGiveXpEvent event = new net.minecraftforge.event.entity.player.PlayerGiveXpEvent(this, p_195068_1_);
++      if (net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(event)) return;
++      p_195068_1_ = event.getAmount();
```

Comment:
Why? The amount is never changed in the event (why not though?).

Reply:
Whoops, I forgot to implement that. I'll add a setter to it